app/controllers/chargeback_controller.rb

```diff
@@ -311,8 +321,13 @@ def cb_rates_delete
         rates.push(params[:id])
       end
       cb_rate = ChargebackRate.find_by_id(params[:id])
-      process_cb_rates(rates, "destroy")  if !rates.empty?
-      add_flash(_("The selected %s was deleted") % ui_lookup(:model=>"ChargebackRate"), :info, true) if ! flash_errors?
+      if cb_rate.default == true
+        add_flash(_("Selected %{model} %{rate} is a default and cannot be deleted") % {:model => "ChargebackRate", :rate  => cb_rate.description}, :error)
+      else
+        process_cb_rates(rates, "destroy")  if !rates.empty?
+        add_flash(_("The selected %s was deleted") % ui_lookup(:model => "ChargebackRate"), :info, true) if !flash_errors?
+      end
+
       self.x_node = "xx-#{cb_rate.rate_type}"
```

Comment:
@GregP i tried to delete a default Chargeback rate from summary screen, Default rate did not get deleted but i was redirected to list view with a correct flash message. Can we move self.x_node = "xx-#{cb_rate.rate_type}" up into the else block so user remains on the default rate summary screen if it doesn't get deleted for some reason.